### DISCUSSION TITLE
Time Stretch Function for Samples

### DISCUSSION THREAD

**Discussion by Korgwald1991** [bangcorrupt/hacktribe#194] [Ideas]
Hello, first of all thank you for your great work. I successfully had my Korg Electribe in Gray motivate a sampler. That makes me super happy and also bucks. My question, is it still possible to implement a time stretch function for samples? I don't know why this feature isn't included by default in the sampler but if nothing major speaks against it you'd make a lot of people happy with it I think :)

**Comment by bangcorrupt** [comment]
May be able to implement an offline time-stretch on the CPU some day.  I can see how samples are passed between the CPU and DSP when they're loaded/saved, could hijack this into a function where we can add whatever processing will fit the resources.  Possibly a re-trigger effect for oldschool tracker style time-stretch too.  It would be cool to re-trigger by dragging across the touch pad.
